spec/auto_scrub_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "autoscrub"
+
+RSpec.describe AutoScrub do
```

Comment:
`AutoScrub` currently has no test coverage, so it might be good to add something. I would consider allowing the autoscrub logger to be injected and then adding some basic integration tests -- does it create a file with the expected output; does it log errors in the place you expect; etc.

Reply:
Added tests that check for output files and lack of exceptions.